apps/sms/style/sms.css

```diff
@@ -302,21 +302,34 @@ form[role="dialog"][data-type="edit"] .bb-tablist > li {
   background: #fff;
 }
 
-.panel:first-child {
+#thread-list {
+  position: absolute;
   left: 0;
+  width: 100%;
+  height: 100%;
+  float: left;
+
+  transition: transform 0.4s ease;
 }
 
-.panel {
+#thread-messages {
   position: absolute;
   left: 100%;
   width: 100%;
   height: 100%;
+  float: left;
 
   transition: transform 0.4s ease;
 }
 
-#main-wrapper[data-position='left'] .panel {
-  transform: translateX(-100%);
+#main-wrapper[data-position="panel-position-1"] {
+  margin-left: 0;
+  float: left;
+}
+
+#main-wrapper[data-position="panel-position-2"] {
+  margin-left: -100%;
```

Comment:
using transformations we'll have nice transitions that are handled by the GPU, which is really smooth on device. If you use margins, this won't be handled by the GPU and it will take CPU and be jerky on a device.